src/Component/Cart.js

```diff
@@ -0,0 +1,38 @@
+/* eslint-disable react/prop-types */
+import React from 'react';
+import Product from './Product';
+
+function Cart(props) {
+  const { cartList } = props;
+  const getTotal = () => {
+    let total = 0;
+    cartList.forEach((product) => {
```

Comment:
@prabhatVinsol use reduce function here and also move this into this component state

Reply:
Done